docs/installation.md

```diff
@@ -35,9 +35,18 @@ Create a file called `${CONTEXT}.xml` in
 necessary, and populate it with the following information:
 
     <Context>
-       <Parameter name="runscanner.configFile" value="/etc/runscanner.json" override="false"/>
+        <Parameter name="runscanner.configFile" value="/etc/runscanner.json" override="false"/>
+        <Parameter name="swagger.baseUrl" value="http://runscanner:8080"/>
     </Context>
 
+Base URL can usually be determined automatically, but may need manual setting if behind a reverse
+proxy, depending on its configuration:
+`swagger.baseUrl=http://runscanner:8080`
+
+If you're deploying Run Scanner to a non-root context, then the property 
+should be:
+`swagger.baseUrl=http://runscanner:8080/${CONTEXT}`
```

Comment:
Remove this. It doesn't change the above comment - it can still be determined automatically in most cases, and can be overridden if needed